source/jormungandr/tests/ptref_tests.py

```diff
@@ -495,3 +495,48 @@ def test_multiple_resource_uri_with_final_collection_uri(self):
         assert 'error' not in response
         stop_areas = get_not_null(response, 'stop_areas')
         eq_(len(stop_areas), 1)
+
+    def test_headsign_stop_time_vj(self):
+        """test basic print of headsign in stop_times for vj"""
+        response = self.query_region('vehicle_journeys?filter=vehicle_journey.name="vehicle_journey 0"')
+        assert 'error' not in response
+        vjs = get_not_null(response, 'vehicle_journeys')
+        eq_(len(vjs), 1)
+        eq_(len(vjs[0]['stop_times']), 2)
+        eq_(vjs[0]['stop_times'][0]['headsign'], "A00")
+        eq_(vjs[0]['stop_times'][1]['headsign'], "vehicle_journey 0")
+
+    def test_headsign_display_info_journeys(self):
+        """test basic print of headsign in section for journeys"""
+        response = self.query_region('journeys?from=stop_point:stopB&to=stop_point:stopA&datetime=20120615T000000')
+        assert 'error' not in response
+        journeys = get_not_null(response, 'journeys')
+        eq_(len(journeys), 1)
+        eq_(len(journeys[0]['sections']), 1)
+        eq_(journeys[0]['sections'][0]['display_informations']['headsign'], "A00")
+
+    def test_headsign_display_info_departures(self):
+        """test basic print of headsign in display informations for departures"""
+        response = self.query_region('stop_points/stop_point:stopB/departures?from_datetime=20120615T000000')
+        assert 'error' not in response
```

Comment:
It's not checked, and not possible to add it, as sometime we return a 200 with an error (no disruptions).